[2018-07-05 08:04:36.968] [ERROR] dateFileLog - { Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
    --------------------
    at Protocol._enqueue (D:\NB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at Protocol.handshake (D:\NB\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at PoolConnection.connect (D:\NB\node_modules\mysql\lib\Connection.js:109:18)
    at Pool.getConnection (D:\NB\node_modules\mysql\lib\Pool.js:45:23)
    at getConnection (D:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:49:12)
    at Object.exec (D:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:116:11)
    at Object.getProject (D:\NB\yujiang.Foil.Node.BizServer\biz\org\processAuthority\processAuthority.db.mysql.js:15:15)
    at buildProject (D:\NB\yujiang.Foil.Node.BizServer\biz\org\processAuthority.js:36:5)
    at fn (D:\NB\node_modules\async\lib\async.js:641:34)
    at Immediate._onImmediate (D:\NB\node_modules\async\lib\async.js:557:34)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3306,
  fatal: true }
[2018-07-05 09:16:52.682] [ERROR] dateFileLog - { Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
    --------------------
    at Protocol._enqueue (D:\NB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at Protocol.handshake (D:\NB\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at PoolConnection.connect (D:\NB\node_modules\mysql\lib\Connection.js:109:18)
    at Pool.getConnection (D:\NB\node_modules\mysql\lib\Pool.js:45:23)
    at getConnection (D:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:49:12)
    at Object.exec (D:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:116:11)
    at Object.getProject (D:\NB\yujiang.Foil.Node.BizServer\biz\org\processAuthority\processAuthority.db.mysql.js:15:15)
    at buildProject (D:\NB\yujiang.Foil.Node.BizServer\biz\org\processAuthority.js:36:5)
    at fn (D:\NB\node_modules\async\lib\async.js:641:34)
    at Immediate._onImmediate (D:\NB\node_modules\async\lib\async.js:557:34)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3306,
  fatal: true }
[2018-07-05 09:17:14.570] [ERROR] dateFileLog - { Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
    --------------------
    at Protocol._enqueue (D:\NB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at Protocol.handshake (D:\NB\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at PoolConnection.connect (D:\NB\node_modules\mysql\lib\Connection.js:109:18)
    at Pool.getConnection (D:\NB\node_modules\mysql\lib\Pool.js:45:23)
    at getConnection (D:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:49:12)
    at Object.exec (D:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:116:11)
    at Object.getProject (D:\NB\yujiang.Foil.Node.BizServer\biz\org\processAuthority\processAuthority.db.mysql.js:15:15)
    at buildProject (D:\NB\yujiang.Foil.Node.BizServer\biz\org\processAuthority.js:36:5)
    at fn (D:\NB\node_modules\async\lib\async.js:641:34)
    at Immediate._onImmediate (D:\NB\node_modules\async\lib\async.js:557:34)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3306,
  fatal: true }
